atoma-state/src/state_manager.rs

```diff
@@ -115,9 +114,12 @@ impl AtomaStateManager {
     /// ```
     #[tracing::instrument(level = "trace", skip_all)]
     pub async fn run(self, mut shutdown_signal: Receiver<bool>) -> Result<()> {
+        dbg!("wakanda");
```

Comment:
Please remove, if it is relevant to have a debugging for this method, I suggest using `tracing` and also in the `instrument` declarative macro, adding the name of the function